Grbl_Esp32/src/I2SOut.cpp

```diff
@@ -74,8 +74,8 @@
 // Reference information:
 //   FreeRTOS task time slice = portTICK_PERIOD_MS = 1 ms (ESP32 FreeRTOS port)
 //
-#    define I2S_SAMPLE_SIZE 4                                     /* 4 bytes, 32 bits per sample */
-#    define DMA_SAMPLE_COUNT I2S_OUT_DMABUF_LEN / I2S_SAMPLE_SIZE /* number of samples per buffer */
+const int I2S_SAMPLE_SIZE = 4; /* 4 bytes, 32 bits per sample */
+const double DMA_SAMPLE_COUNT = I2S_OUT_DMABUF_LEN; / I2S_SAMPLE_SIZE /* number of samples per buffer */
```

Comment:
The ; is in the wrong place.  It should be after / I2S_SAMPLE_SIZE 

Changing that fixes compilation errors with the 6_pack machines